apps/dashboard/src/components/primitives/dialog.tsx

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+import * as DialogPrimitive from '@radix-ui/react-dialog';
+import { Cross2Icon } from '@radix-ui/react-icons';
+
+import { cn } from '@/utils/ui';
+import { cva, VariantProps } from 'class-variance-authority';
+
+const Dialog = DialogPrimitive.Root;
+
+const DialogTrigger = DialogPrimitive.Trigger;
+
+const DialogPortal = DialogPrimitive.Portal;
+
+const DialogClose = DialogPrimitive.Close;
+
+const DialogOverlay = React.forwardRef<
+  React.ElementRef<typeof DialogPrimitive.Overlay>,
+  React.ComponentPropsWithoutRef<typeof DialogPrimitive.Overlay>
+>(({ className, ...props }, ref) => (
+  <DialogPrimitive.Overlay
+    ref={ref}
+    className={cn(
+      'data-[state=open]:animate-in data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=open]:fade-in-0 fixed inset-0 z-50 bg-neutral-950/10',
```

Comment:
`bg-neutral-100`  ?

Reply:
it doesn't have alpha, but even the alpha version is not the same as what is in the Figma, which is `hsla(222, 32%, 8%, 0.1)` but this `bg-neutral-950/10` doesn't produce the same result but really close :(